yaqd_wright/_wright_filter_wheel.py

```diff
@@ -1,21 +1,31 @@
 __all__ = ["WrightFilterWheel"]
 
 import asyncio
+from typing import Dict
 import time
+
 from yaqd_core import ContinuousHardware, DiscreteHardware, aserial
+from ._serial import SerialDispatcher
```

Comment:
You do not need this line, all other references to SerialDispatcher should be `aserial.ASerial`